azure/vars.tf

```diff
@@ -3,9 +3,9 @@ variable "region" {
   default     = "centralus"
 }
 
-variable "zone" {
+variable "availability_zone" {
   description = "Availability Zone"
-  default     = null
+  default     = []
```

Comment:
Why default this to no AZs, while we have multi-azs defined in the other clouds?